about.html

```diff
@@ -0,0 +1,128 @@
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+                      </ul>
+    </div>
+    <header id="header">
+        <div class="logo"><a href="index.html">
+          <img src="images/gorilla-logo.png" alt="logo"></a>
+        </div>
+        <nav id="nav-bar" class="nav-bar d-none">
+          <div id="close-icon" class="close-icon d-none">
+            <i class="fas fa-times-circle"></i>
+          </div>
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="index.html" >Home</a>
+            <a class="nav-link" href="#" >Program</a>
+            <a class="nav-link" href="#" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+        </div>
+      </header>
+    <div class="banner">
+        <div class="hidden_a">
+        <div class="hidden_b">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals and Events</h2>
+        <p>Better known as ‘Hillywood’, the Rwanda Film Festival has been bringing films from around Rwanda, Africa and the world to Kigali every July for more than 10 years now</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+        <h5>Please contact us per Email for any futher questions about Visit Rwanda 2021! <br> <b>mucyosofa@gmail.com</b></h5>
+            </div>
+         </div>
+    </div>
+
+    <section class="main">
+        <h2>Visit Rwanda Now</h2>
+        <div class="hr">
+         </div>
+         <div class="global">
+           <h3>Rwanda is now open. Learn more here about Virunga Nation Park.</h3>  
+         </div>
+
+         <div class="imagelogo"><img src="images/gorilla-logo.png" alt="logo"></div>
+
+         <div class="lineb"></div>
+         <div class="rda">
+             <h2>Rwanda lies just south of the Equator</h2></div> 
+             <div class="hr"></div>
+             
+         <div class="view">
+             <h6>View special offers from Rwandan tour and travel operators here.</h6>
+         </div>
+         <div class="dance">
+            <h1>2021 <br> Tradition Dance</h1> 
+            
+           </div>
+         <div class="gorila">
+             <h3>2020 <br> Nation Park</h3>  
+         </div>
+        </section>
+    <footer>
+      <div class="partner" id="sponsor">
+        <h2>Partner</h2>
+    
+        <div id="sponsors">
+        <div id="sponsor_1">mozilla</div>
+        <div id="sponsor_2">Google</div>
+        <div id="sponsor_3">NAVER</div>
+        <div id="sponsor_4">daumkakao</div>
+        <div id="sponsor_5">NAVER</div>
+    </div></div>
```

Comment:
- [ ] In the partner's section, try as much as possible to make sure that when the text is wrapping, the two last words should be aligned at the center and not to the left.

Reply:
- [ ] As I said in the first review, you have to make sure that when the text is wrapping, the 2 last words should be center and not on the sides. Please kindly apply this change on both pages.